### PR TITLE
Initial import of source code.

### PR DESCRIPTION
**Pull request by pirmink**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+the presence of blank separating them - just the way any halfway decent
+programmer would write code anyways.
+
+I have not written that it converts FORTRAN77 to Fortran 90 on purpose - the
+language standard and the format of the source files are two different
+things. Not completely unrelated, but many programmers started using new Fortran
+features while still writing fixed-format source code.
+
+Authors of ffixed2free: Pirmin Kaufmann and Martin Schraner, MeteoSwiss (at the
+time of writing the code).
```
Comment (cosunae) [inline README.md:25]: Can you add few instructions on requirements, how to build it and how to run it?

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	    	sh '''#!/bin/bash
+     	    	cd src
+	    	make
+                ''' 
+            }
+        }
+        stage('Test') { 
+            steps {
+                sh '''
+		./src/ffixed2free
```
Comment (cosunae) [inline jenkins/Jenkinsfile]: I think the tests should do test the output, not only run it. 
you could have a collection of small examples to test, together with a reference, which is the expected output. Then you could have a test.py that would basically run all the examples and compare the output produce against the reference (with a simple diff)

**Review by cosunae — CHANGES_REQUESTED** [review]

File: jenkins/Jenkinsfile (hunk 2 above)
Comment (pirmink) [reply jenkins/Jenkinsfile]: Added FORTRAN 77 example file for testing, wich is now use in the 'Test' stage of Jenkins.

File: README.md (hunk 1 above)
Comment (pirmink) [reply README.md:25]: Added compiling an usage instructions in README.md

**Review by cosunae — APPROVED** [review]
looks good to me. Thanks for implementing the changes

**Pull request merged**